code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -68,6 +68,7 @@ var/list/robot_verbs_default = list(
 	var/lockcharge //Used when locking down a borg to preserve cell charge
 	var/speed = 0 //Cause sec borgs gotta go fast //No they dont!
 	var/scrambledcodes = 0 // Used to determine if a borg shows up on the robotics console.  Setting to one hides them.
+	var/pdahide = 0 //Used to hide the borg from the messenger list

```

Comment:
`FALSE`

Reply:
That wouldn't be consistent with the file though.

Reply:
True, but we should ideally be working towards better standards when introducing newer code.

Reply:
This is a bugfix solely, not a standard rewrite. I rather not complicate any merge potential with semantic Boolean values. Though, thanks for the reviewing in general. I do appreciate the thought.